02-when-vector-data-dont-line-up-CRS.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: "Lesson 02: When Vector Data Don't Line Up - Handling Spatial Projections 
-- CRS - in R"
+title: "Lesson 02: When Vector Data Don't Line Up -- Handling Spatial
```

Comment:
@lwasser Does NEON or do we have a preferred style guide to follow for writing?  It would help me for knowing which grammatical edits to make.